tron/core/actionrun.py

```diff
@@ -918,6 +925,7 @@ def __init__(self, action_graph, run_map):
                 proxy.attr_proxy('is_scheduled', any),
                 proxy.attr_proxy('is_cancelled', any),
                 proxy.attr_proxy('is_active', any),
+                proxy.attr_proxy('is_waiting', all),
```

Comment:
this is true only if all runs are in `waiting`, is this the logic we need?

Reply:
Right, changed to `any`. That way it's "waiting" in the following case:
  action1: succeeded
  action2: waiting